src/Core/src/Platform/Android/ImageViewExtensions.cs

```diff
@@ -17,6 +17,11 @@ public static void Clear(this ImageView imageView)
 
 		public static void UpdateAspect(this ImageView imageView, IImage image)
 		{
+			if (image.Aspect is Aspect.AspectFill)
+				imageView.SetAdjustViewBounds(false);
+			else
+				imageView.SetAdjustViewBounds(true);
```

Comment:
The changes are impacting other related Image UITests: 
**Issue18242Test**
![image](https://github.com/user-attachments/assets/5adf4371-14fc-45cf-a131-322f67cd15a0)

Reply:
I’ve modified the fix and rerun the mentioned test. The test passed successfully without any issues.
Also I have committed the same. Could you please check and let me know.